src/components/styles/FormContainer.styled.ts

```diff
@@ -1,6 +1,7 @@
 import styled from 'styled-components'
 
-export const FormContainer = styled.form`
+export const FormContainer = styled.form<{ noValidate?: true }>`
```

Comment:
does this mean that the noValidate arg is always true?

Reply:
this is to set noValidate to be true by default, becos since we're using react-hook-form we don't need default browser vqalidation anymore